### ISSUE TITLE
Add explanations to all panels on the Prombench dashboard

### ISSUE THREAD

**Issue by gerhard** [prometheus/test-infra#305]
It's difficult to understand what the different panels mean, and how they correlate. It would be great to have more information on them, similar to https://grafana.com/grafana/dashboards/10991

@codesome has all the context

**Comment by geekodour** [comment]
Yes, I think having more info will be very useful as well.

@krasi-georgiev any updates on how to proceed with #186 , it seems these are somewhat related.

**Comment by krasi-georgiev** [comment]
yes this would a good improvement and need some feedback from Prometheus users what metrics are useful and which aren't and yes having more info is always helpful. Just need someone to spend the time to do the research and revisit all panels or update to the latest dashboard from Brian's collections if these are updated there.
